packages/contracts-bedrock/src/L2/interfaces/IMintableAndBurnableERC20.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
Only if it is gonna be used on the `SuperchainERC20Bridge` contract
```suggestion
pragma solidity =0.8.25;
```

Reply:
It is also being used in the `L2StandardBridgeInterop` that is `0.8.15` so it needs to be open. Also all interfaces need to be like this to support integrations, at least while there are differents pragmas across the code